.gitignore

```diff
@@ -179,3 +179,6 @@ ENV/
 
 # pip wheel dist info
 pip-wheel-metadata/
+
+# vscode
+.vscode/
```

Comment:
This certainly does not belong to this PR. https://julien.danjou.info/properly-managing-your-gitignore/.

Reply:
Done.